test/strategy.test.ts

```diff
@@ -1,9 +1,11 @@
-import { getAddress } from '@ethersproject/address';
 import { performance } from 'perf_hooks';
 import fetch from 'cross-fetch';
 import snapshot from '../src';
 import snapshotjs from '@snapshot-labs/snapshot.js';
 import addresses from './addresses.json';
+import starknetAddresses from './addresses-starknet.json';
+
+const STARKNET_CHAIN_IDS = ['0x534e5f4d41494e', '0x534e5f5345504f4c4941'];
```

Comment:
Maybe we should get from networks.json as well

Reply:
Best is to add a new param here https://github.com/snapshot-labs/snapshot.js/blob/master/src/networks.json#L1974-L2002 like `starknetNetwork: true` and import here 

or last option is to use `s` and `s-tn` keys to filter